src/pages/references/language-clarity-functions.md

```diff
@@ -0,0 +1,11 @@
+---
+title: Clarity Functions
```

Comment:
with the new section the nav shows: "clarity language > clarity <section>". Maybe we can update the titles of the individual pages and remove `Clarity `, so the nav doesn't display the word clarity twice?